src/test/unittest/tools.py

```diff
@@ -95,9 +95,39 @@ def _get_dev_info(self, dev_path):
                               .format(dev_path))
         return dev
 
+    # for ndctl v63 we need to parse ndctl list in a different way than for v64
+    def _get_dev_info_63(self, dev_path):
+        dev = None
+        devtype = 'chardev'
+        daxreg = 'daxregion'
+
+        for d in self._get_ndctl_list_output('list', '-v'):
+            if daxreg in d:
+                devices = d[daxreg]['devices']
+                for device in devices:
+                    if devtype in device and \
+                            os.path.join('/dev', device[devtype]) == dev_path:
+                        # only params from daxreg are intrested at this point,
+                        # other values are read by _get_dev_info() earlier
+                        dev = d[daxreg]
+
+        if not dev:
+            raise futils.Fail('ndctl does not recognize the device: "{}"'
+                              .format(dev_path))
+
+        return dev
+
     def _get_dev_param(self, dev_path, param):
+        p = None
         dev = self._get_dev_info(dev_path)
-        return dev[param]
+
+        try:
+            p = dev[param]
+        except KeyError:
+            dev = self._get_dev_info_63(dev_path)
+            p = dev[param]
+
```

Comment:
Shouldn't we use ndctl --version to select correct function instead of handling an exception?